docs/extend/extend-apiml/api-mediation-caching-service.md

```diff
@@ -0,0 +1,133 @@
+# Using the Caching Service
```

Comment:
@nannanli these new files `extend-apiml/api-mediation-catching-service` & `extend-apiml/api-mediation-redis` added in this [commit](https://github.com/zowe/docs-site/pull/1725/commits/96bd7fb1c7f0763f2f91344b86444c74f47a9f82) needs to be referenced in the [sidebar.js](https://github.com/zowe/docs-site/blob/zowe-docs-v2/sidebars.js#L178)